app/views/layouts/admin.html.erb

```diff
@@ -32,6 +32,7 @@
   data-controller="snackbar"
   data-action="snackbar:add@document->snackbar#addItem">
   <header class="crayons-header">
+    <span id="route-change-target" tabindex="-1"></span>
```

Comment:
i18n (potentially blocking): Since this was changed in the top bar template, change it here as well to use `<%= t("views.main.header.skip") %>`. The only question I have about this is should the admin skip link have it's own `<%= t("views.admin.header.skip") %>`. If the latter is the case, we can table this and add it to i18n work for the admin experience. cc: @Ridhwana

Reply:
I guess since the style/function of the skip link is consistent across the whole app it probably makes sense for the text to come from the same i18n string 🤔  I'll wait for Ridhwana's thoughts though

Reply:
> The only question I have about this is should the admin skip link have it's own <%= t("views.admin.header.skip") %>

I'm leaning towards the admin having it's own skip link, that will form as part of it's own admin header object. However, rather than pre-emptively creating this I think let's keep it reading from `main` for now and when we tackle the i8n on the admin we can make an informed decision.

Reply:
I'm leaning towards the admin having it's own skip link, that will form as part of it's own admin header object. However, rather than pre-emptively creating this I think let's keep it reading from main for now and when we tackle the i8n on the admin we can make an informed decision.

Reply:
Done 🚀  Thanks both!